include/bluetooth/conn.h

```diff
@@ -81,10 +81,29 @@ struct bt_conn_le_phy_info {
 	u8_t rx_phy; /** Connection receive PHY */
 };
 
+/** Connection PHY options */
+enum {
+	/** Convenience value when no options are specified. */
+	BT_CONN_LE_PHY_OPT_NONE = 0,
+
+	/** LE Coded using S=2 coding preferred when transmitting. */
+	BT_CONN_LE_PHY_OPT_CODED_S2  = BIT(0),
+
+	/** LE Coded using S=8 coding preferred when transmitting. */
+	BT_CONN_LE_PHY_OPT_CODED_S8  = BIT(1),
+
+	/** No preference for transmit PHY, pref_rx_phy is ignored. */
```

Comment:
Should we rather let `pref_tx_phy = 0` indicate that there is no preference for transmit PHY? Similar for RX.

Reply:
@carlescufi @jhedberg Any thoughts on removing the two "no preference" options, and instead have `BT_CONN_LE_PHY_OPT_NONE = 0`, mean no preference? I'm a bit indicisive, I think i like this suggestion better.

Reply:
none == no preference sounds intuitive and simpler to me